src/client/cl_screen.c

```diff
@@ -387,6 +387,10 @@ void Cl_UpdateScreen(void) {
 		}
 	}
 
+	if (cls.state == CL_ACTIVE) {
+		R_AddStains();
+	}
+
```

Comment:
Eeeeeh. This is really ugly here. Is there a specific reason this can't be encapsulated in R_DrawView?